k8s guestbook application example for use in tutorial (#626)
radius-project/samples: samples/README.md

# Reference applications

> Reference apps are templates that show a complete app. You are able to clone and deploy any of these apps to try out various Project Radius functionality.

Reviewer: ```suggestion
Clone and deploy reference applications to learn more about Radius..
```

> Reference apps are templates that show a complete app. You are able to clone and deploy any of these apps to try out various Project Radius functionality.

Reviewer: ```suggestion
Clone and deploy reference applications to learn about Radius.
```

> Reference apps are templates that show a complete app. You are able to clone and deploy any of these apps to try out various Project Radius functionality.

Reviewer: ```suggestion
Clone and deploy reference applications to learn more about Radius.
```
